presidio-anonymizer/setup.cfg

```diff
@@ -2,4 +2,5 @@
 max-line-length = 88
 docstring-convention = numpy
 per-file-ignores =
-    tests/*: D
\ No newline at end of file
+    tests/*: D
+extend-ignore = E203 D100 D202 ANN101 ANN102 ANN204 ANN203
```

Comment:
Aligned with anonymizer, sorry for pushing this there, don't wan't to disable my pre-commit

Reply:
Can you please specify what are those ignores do in a remark?